cli/src/fail_job.rs

```diff
@@ -0,0 +1,50 @@
+use async_trait::async_trait;
+use color_eyre::eyre::Result;
+
+use crate::ExecuteZeebeCommand;
+use clap::Args;
+
+use zeebe_client::{
+    api::{FailJobRequest, FailJobResponse},
+    ZeebeClient,
+};
+#[derive(Args)]
+pub(crate) struct FailJobArgs {
+    // the unique job identifier, as obtained when activating the job
```

Comment:
:question: Do we want to copy the comments from proto file, or do we want to remove them. It is redundant code, but could also be helpful

Reply:
I don't have an opinion on this :shrug: